lib/puppet/functions/to_yaml.rb

```diff
@@ -2,21 +2,28 @@
 # @summary
 #   Convert a data structure and output it as YAML
 #
-# @example how to output YAML
+# @example How to output YAML
 #   # output yaml to a file
 #     file { '/tmp/my.yaml':
 #       ensure  => file,
 #       content => to_yaml($myhash),
 #     }
+# @example Use options control the output format
+#   file { '/tmp/my.yaml':
+#     ensure  => file,
+#     content => to_yaml($myhash, {indentation: 4})
```

Comment:
`{indentation: 4}` doesn't look like a valid _puppet_ hash to me.  Or am I missing something?